packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1318,6 +1319,9 @@ export const components: ThemeOptions["components"] = {
             transform: "none",
           }),
         }),
+        "& > .MuiTypography-root": {
+          lineHeight: `${theme.typography.ui.lineHeight}rem`,
```

Comment:
Pref for no `rem` here, unitless `line-height`s are 👍🏼

Reply:
This was the issue — when unitless, the lineHeight becomes `17.4px`. With the unit, it's `16px` (as intended).

Reply:
Ah, I see. `Typography.root` is setting the `body1` `line-height` here, which is slightly taller than the `ui` `line-height` we use for the label.

`lineHeight: "unset"` should ensure that the label defines our height without us needing to match them manually.

Reply:
Fixed!